pkg/binding/nested/field_copy.go

```diff
@@ -0,0 +1,23 @@
+package nested
+
+import (
+	"bytes"
+	"strings"
+
+	"k8s.io/client-go/util/jsonpath"
+)
+
+func GetNested(obj map[string]interface{}, path_segments ...string) (string, bool, error) {
+	path := "{." + strings.Join(path_segments, ".") + "}"
+	j := jsonpath.New("")
+	err := j.Parse(path)
+	if err != nil {
+		return "", false, err
+	}
+	buf := new(bytes.Buffer)
+	err = j.Execute(buf, obj)
```

Comment:
using j.FindResults(obj) is more appropriate, because jsonpath can return multiple results that could be then converted into individual bindings by the caller. Using `Execute` always return just single result.

Having said that, this function should return a pair of `[][]reflect.Value, error` to reflect that.